src/web/docusaurus/docs/api-services/auth.md

```diff
@@ -1,3 +1,86 @@
+---
+sidebar_position: 5
+---
+
 # Authentication and Authorization
 
-Documentation in construction...
+## Summary
+
+SSO service is required for authentication. Telescope doesn't store user information itself, we just send it to Seneca's SSO. From that interaction, Telescope creates a token that will only work for a limited amount of time. The token is used in authenticating in various Telescope services, without a need to make requests to Seneca's servers every time. The same token system is also used throughout our Supabase back end.
+
+## Stages
+
+_This is what happens step by step in the SSO service:_
+
+- Request to log in is sent with the user entering their log in information into Telescope's front end.
```

Comment:
Maybe we can reword it to: `The action of logging in to the Telescope website will send a request to login.`

Reply:
Sure. I think your suggestion could also be rewritten better, but it is already a step up to the previous version.